app/models/user.rb

```diff
@@ -575,6 +623,74 @@ def flipper_id
 
   private
 
+  # rubocop:disable Metrics/BlockLength
+  def migrate_users_fields_to_users_settings(users_setting_record)
```

Comment:
We can really simplify this function by generalizing the code a little.We'll tackle it when we pair today.